components/Layout/HeaderMobile.tsx

```diff
@@ -0,0 +1,34 @@
+import Link from 'next/link';
+
+const HeaderMobile = (): JSX.Element => {
+  return (
+    <header className={'bg-blue-500 p-4'}>
```

Comment:
Du brauchst die curly brackets nicht wenn du einen string als prop übergibst.

```suggestion
    <header className='bg-blue-500 p-4'>
```